application/templates/admin/data_sources.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base.html" %}
+{% from "_shared/_breadcrumb.html" import breadcrumb %}
+
+{% set breadcrumbs =
+  [
+    {"url": url_for("static_site.index"), "text": "Ethnicity facts and figures"},
+    {"url": url_for('admin.index'), "text": "Admin"},
+  ]
+%}
+
+{% block pageTitle %}Data sources{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">Data sources</h1>
+
+          <ul class="govuk-list">
+            {% for data_source in data_sources %}
```

Comment:
Is there any reusable component we can turn this into, considering we have two views doing very similar things?

Reply:
Possibly - but I have a hunch they might need to show different information - and the admin view will become part of a radio button list, so the HTML will be different.